templates/webhook.bin.erb

```diff
@@ -168,7 +168,16 @@ class Server < Sinatra::Base
 
     # When a branch is being deleted, a deploy against it will result in a failure, as it no longer exists.
     # Instead, deploy the default branch, which will purge deleted branches per the user's configuration
-    deleted = data['deleted'] rescue false
+    deleted = (
+      data['deleted'] rescue false || # Github
+      data['push']['changes'][0]['closed'] rescue false || # Bitbucket
```

Comment:
Bitbucket *server* (or stash) needs to look for `data['refChanges'][0]['type'] == 'DELETED'`
See https://confluence.atlassian.com/bitbucketserver/post-service-webhook-for-bitbucket-server-776640367.html